### DISCUSSION TITLE
[Feature Request] Key binds, changing all of them (also, WASD movement)

### DISCUSSION THREAD

**Discussion by BloodyIron** [OpenTTD/OpenTTD#11730] [Features]
## Version of OpenTTD  v1.11.0

I'm all upons the classic isometric games like this. But quite frankly, I've been spoiled by games like Stellaris and others, where wasd by default lets me move around the view area, instead of randomly selecting functions I don't know what they do.

While I suspect there's a significant interest in generally keeping the feel of OpenTTD true to the classic, I really would like to see a compromise struck whereby we can change all the key binds in the game, and hopefully have a "wasd" equivalent key bind profile we can switch to out of the box, for those of us that want half new age, half classic experience.

If the key bind ability exists, well... I really could not find it. I searched all over the settings (and by the way, click holding to find menus for things threw me off for a while too).

Any chance we can add to a mailestone/todo list in some way please? :) I know this isn't likely to be high priority, but it really would improve the UX.

**Comment by nielsmh** [comment]
It exists but there is no UI for it. You can edit `hotkeys.cfg` in your user settings folder. (On Windows that's Documents\OpenTTD.)

**Comment by ldpl** [comment]
You can't bind map movement in hotkeys.cfg because it requires keyup/down and hotkeys only allow keypress atm.

**Comment by BloodyIron** [comment]
Sounds like two areas for improvement. Increasing what can be rebound, and UI for it ;)

**Comment by HugoCortell** [comment]
I agree
